raiden/network/pathfinding.py

```diff
@@ -568,6 +569,19 @@ def query_address_metadata(
     if response.status_code != 200:
         raise PFSReturnedError.from_response(response_json)
 
+    dummy_route_metadata = RouteMetadata(
```

Comment:
Nitpick: I don't like the naming, since it not really a placeholder or something, but rather an object created for validation. What about simply `route_metadata` and adding a comment that it's only used for validation?